style.css

```diff
@@ -122,7 +167,7 @@ body {
   background: #fff;
   border: 1px solid #def1e6;
   border-radius: 16px;
-  width: 90%;
+  width: 100%;
   height: 552px;
 }
```

Comment:
- [ ]  To create a user-friendly UI and improve the user experience please make sure that your work cards are fully responsive across all screen sizes. Your work card looks really good but it's not responsive, Please make sure your work card is responsive and can be viewed across all screen sizes. To fix this issue consider the hint below:
`Note`: This is an issue found during screen resize, below `768px`.
  - [ ] Remove the specified `width` and `height` you have set for the card container. As this will help make the content of the card span the full width of its container by default.
  
  `Important note`: **These changes should also be implemented on the class name `work-border-two` as well.**